app/models/podcast.rb

```diff
@@ -67,6 +67,19 @@ def default_feed
     super || build_default_feed(podcast: self, private: false)
   end
 
+  def apple_config
+    if defined?(@apple_config)
+      @apple_config
+    else
+      @apple_config = Apple::Config.where(feed_id: feeds.pluck(:id)).first
```

Comment:
wait, but that's looking for the configs based on the first feed's id?
but the config is associated with the private feed now, that won't be the first feed, the default public feed will be.
This should be looking for the apple private feed id, not just the first in the list, right?

Like, this should find the `feed.publish_apple? == true`, not just the first one?

Reply:
nvm, cleared this up in slack, my bad.